llvm/test/Analysis/LoopAccessAnalysis/different-strides-safe-dep-due-to-backedge-taken-count.ll

```diff
@@ -80,13 +76,8 @@ exit:
 define void @unknown_dep_known_safe_due_to_backedge_taken_count(ptr %A) {
 ; CHECK-LABEL: 'unknown_dep_known_safe_due_to_backedge_taken_count'
 ; CHECK-NEXT:    loop:
-; CHECK-NEXT:      Report: unsafe dependent memory operations in loop. Use #pragma clang loop distribute(enable) to allow loop distribution to attempt to isolate the offending operations into a separate loop
-; CHECK-NEXT:  Unknown data dependence.
+; CHECK-NEXT:      Memory dependences are safe
```

Comment:
This might be wrong due to my previous note: If the backedge taken count is not known then there is no safe distance between the pointers (unless the stride is 0, diferent from not computable)

Reply:
OK, I was confused here why a previously unsafe dep became safe when the title says `unknown_dep_known_safe_due_to_backedge_taken_count`. I think you specifically added this test case in d5f2753067df89ed4c49d387deb0b3a6b59f8175 to fix here.

Reply:
Yes exactly, the test was specifically added for this PR, although I should probably have added a TODO....